.coveragerc

```diff
@@ -1,5 +1,13 @@
 ; http://nedbatchelder.com/code/coverage/config.html
 
 [run]
-include = st2*
-omit = *tests/*
```

Comment:
Don't we need to omit the coverage generation for the `tests/` files themselves?

Reply:
The second bullet point in the PR summary:

* Do we care about coverage in `*/tests/`? Some projects like it ([pynacl](https://github.com/pyca/pynacl/pull/290)), some people don't ([rebar3](https://github.com/erlang/rebar3/issues/1635)).

I used to not include tests in coverage results, but the `pynacl` project changed my mind. I have found dead tests in projects thanks to coverage results. So I'm in favor of including `*tests/` in coverage results.